sk0-back/src/utils/mailer.service.ts

```diff
@@ -0,0 +1,29 @@
+import * as nodemailer from "nodemailer";
+
+export class MailerService {
```

Comment:
I think we should create a generic notificationService rather (or even more maybe), since this email notification (or messaging) should be only one implementation of it.

Might wanted to switch later to another package or whatsapp/telegram and we probably should use an interface with the delivery of the message.

Reply:
This is done too.